src/pmfs/pmfs_cmd.c

```diff
@@ -0,0 +1,742 @@
+/**
```

Comment:
The whole file can be merged to `vos_target_fs.c` and remove the use of prefix "pmfs_", because we use "pmfs_" as the public APIs in `pmfs.c`

Reply:
ACK, done